crates/typst/src/text/smartquote.rs

```diff
@@ -233,6 +233,7 @@ impl<'s> SmartQuotes<'s> {
             "no" | "nb" | "nn" if alternative => low_high,
             "ru" | "no" | "nb" | "nn" | "ua" => ("’", "’", "«", "»"),
             "gr" => ("‘", "’", "«", "»"),
+            "he" => ("׳", "׳", "״", "״"),
```

Comment:
Don't believe the preview, the order is correct: single and then double quotes.

Reply:
In this case, maybe it's better to use `\u{..}` notation to not trigger BiDi weirdness.

Reply:
Done.